src/main/resources/db/migration/V7__ISI-889_verlinkung_e_akte.sql

```diff
@@ -0,0 +1,15 @@
+--
+-- Neues Feld "eAkte" bei der Abfrage (ISI-889).
+--
+BEGIN;
```

Comment:
Da es sich bei URIs in der EAkte um Deeplinks handelt können diese sehr lange sein.

Die Länge sollte auf mindestens 8000 Octets (characters) erhöht werden um dem folgenden RFC zu entsprechen.

https://www.rfc-editor.org/rfc/rfc9110#section-4.1-5